src/SearchButton.cc

```
@@ -32,22 +32,27 @@ SearchButton::SearchButton(Decoration *decoration, const int buttonIndex, QObjec
     //painter->setRenderHint(QPainter::Antialiasing, true);
     setPenWidth(painter, 1.25);
 
-    const qreal circleRadius = 4.0;
-    const QPointF circleCenter(-2, -2);
-    painter->drawEllipse(circleCenter, circleRadius, circleRadius);
+    const QPointF circleTopLeft = snapper.snap(QPointF(-6.0, -6.0));
+    const QPointF circleBottomRight = snapper.snap(QPointF(2.0, 2.0));
+    const QRectF circleRect(circleTopLeft, circleBottomRight);
+    painter->drawEllipse(circleRect);
 
+    const qreal sqrt2 = std::sqrt(2.0);
```

Clang (14.0.6)

[warning] variable 'sqrt2' is not initialized

(cppcoreguidelines-init-variables)